pkg/admission/interfaces.go

```diff
@@ -36,4 +36,8 @@ type Attributes interface {
 type Interface interface {
 	// Admit makes an admission decision based on the request attributes
 	Admit(a Attributes) (err error)
+
+	// Handles returns true if this admission controller can handle the given operation
+	// where operation can be one of CREATE, UPDATE, DELETE, or CONNECT
```

Comment:
Add constants for the enumerated set of options so its clear in the long term what operations are supported in Kubernetes.  I think it's ok for it to remain a string as part of this PR, I just want a constant to find all future references, and this PR seems like a good one to do it.